exercises/verify.cpp

```diff
@@ -9,10 +9,22 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+template<typename T>
+void find(T A[],short N, T val){
+	bool found = false;
+	for(int i=0; i<N; i++){
+		if(A[i] == val){
+		found = true;
+		cout << "Element Found" << endl;
+		return;
+		}
+	}
+	cout << "Element Not Found" << endl;
```

Comment:
a che serve la variabile `found`?

inoltre, potresti mettere un po' di spazi dentro l'ife ed aumentare la leggibilità

Comunque l'idea della funzione con il template non è affatto male 👍

Reply:
un refuso, prima avevo fatto la variabile found che serviva come condizione di uscita dal ciclo for... c'era un and logico poi ho pensato ma si può fare molto più semplice xD